dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -132,6 +132,47 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         m_notificationHandlers.remove(token);
     }
 
+    HRESULT __stdcall AppNotificationManager::Activate(
```

Comment:
If the developer subscribes to the first even handler after the REgister API call we should THROW.

Reply:
There was no other place to put this comment, so i put it here :)